### DISCUSSION TITLE
Comparison between other graphics API?

### DISCUSSION THREAD

**Discussion by dellwatson** [gpuweb/gpuweb#4189] [Q&A]
since now we can use gpu on web, how does performance compared to other graphic API like vulkan? 
would webgpu have same gpu usage in efficiency and optimisation ?

**Comment by aldrians** [comment]
DirectX, vulkan, opengl, metal are api for native app, webgpu api for browser (also have wgpu to bring webgpu outside browser).

**Comment by Frendysemon** [comment]
Yeah WebGPU should be pretty close to Vulkan for most games. You lose some low level control but the performance gap should be small once the implementations mature.
